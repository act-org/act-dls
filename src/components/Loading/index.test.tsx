/**
 * @prettier
 */

import * as React from 'react';

import render from '../../helpers/test/render';

import { Loading } from '.';

describe('LoadingPrimary', () => {
  it('matches the snapshot', () => {
    const { container } = render(<Loading title="Loading..." />);

    expect(container).toMatchSnapshot();
  });
});
