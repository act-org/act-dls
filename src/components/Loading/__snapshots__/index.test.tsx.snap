// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPrimary matches the snapshot 1`] = `
<div>
  <div
    class="makeStyles-container-1"
    data-testid="loading"
  >
    <div
      class="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
      role="progressbar"
      style="width: 24px; height: 24px;"
    >
      <svg
        class="MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
          cx="44"
          cy="44"
          fill="none"
          r="19.75"
          stroke-width="4.5"
        />
      </svg>
    </div>
    <span
      class="MuiTypography-root makeStyles-typographyRoot-2 MuiTypography-overline"
    >
      Loading...
    </span>
  </div>
</div>
`;
